cmd/zetaclientd/start.go

```diff
@@ -234,6 +229,40 @@ func start(_ *cobra.Command, _ []string) error {
 		masterLogger.Info().Msg("TSS listener received an action to shutdown zetaclientd.")
 		signalChannel <- syscall.SIGTERM
 	})
+	// debug: printout connected peers
+	go func() {
+		for {
+			time.Sleep(30 * time.Second)
+			ps := server.GetKnownPeers()
```

Comment:
from go-tss:

```
func (t *TssServer) GetKnownPeers() []peer.AddrInfo {
	var infos []peer.AddrInfo
	host := t.p2pCommunication.GetHost()

	for _, conn := range host.Network().Conns() {
		p := conn.RemotePeer()
		addrs := conn.RemoteMultiaddr()
		pi := peer.AddrInfo{
			p,
			[]maddr.Multiaddr{addrs},
		}
		infos = append(infos, pi)
	}
	return infos
}
```
I wonder if there are any scenarios in which there could be multiple connections to the same peer?

Reply:
i'd say unlikely, but i'm not sure. this will return all.